src/go/capture/buffer/buffer.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package buffer
```

Comment:
nit: i would probably put this in the `capture` package so it's `capture.Buffer`, but this is fine also.

generally, i'm not a fan of "micropackages", just like I'm not a fan of microservices :)